code/modules/spells/spell_types/construct_spells.dm

```diff
@@ -63,6 +63,23 @@
 
 	summon_type = list(/turf/closed/wall/mineral/cult/artificer) //we don't want artificer-based runed metal farms
 
+/obj/effect/proc_holder/spell/aoe_turf/conjure/door
+	name = "Summon Cult Door"
+	desc = "This spell constructs a cult Airlock."
+
+	school = "conjuration"
+	charge_max = 300
+	clothes_req = FALSE
+	invocation = "none"
+	invocation_type = "none"
+	invocation_time = 50
+	range = 0
+	action_icon = 'icons/mob/actions/actions_cult.dmi'
+	action_icon_state = "airlockconstruct"
+	action_background_icon_state = "bg_cult"
+
+	summon_type = list(/obj/machinery/door/airlock/cult)
```

Comment:
As with the one above, you can farm runed metal by just spamming the doors and breaking

Reply:
i tested what would happen if i break one of the airlock and turns out it will give you normal metal